Makefile

```diff
@@ -0,0 +1,17 @@
+.DEFAULT_GOAL := help
+.PHONY := requirements
+
+# Generates a help message. Borrowed from https://github.com/pydanny/cookiecutter-djangopackage
+help: ## Display this help message
+	@echo "Please use \`make <target>' where <target> is one of"
+	@perl -nle'print $& if m{^[\.a-zA-Z_-]+:.*?## .*$$}' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m  %-25s\033[0m %s\n", $$1, $$2}'
+
+requirements:
+	pip install -r requirements.txt
+
+upgrade: ## update the pip requirements files to use the latest releases satisfying our constraints
+	docker-compose build && docker-compose run hermes make upgrade-requirements
+
+upgrade-requirements: export CUSTOM_COMPILE_COMMAND=make upgrade
+upgrade-requirements: ## update the pip requirements files to use the latest releases satisfying our constraints, outside of docker
```

Comment:
I'm a little confused by the ```outside of docker``` comment. I would think that means it's run outside a container, but upgrade-requirements is called by upgrade in a docker-compose command, so I'm assuming this means it's called from outside the container to run inside the container. Am I understanding correctly?

Reply:
It's confusing what I meant is that it is not strictly tied to docker, and could be run outside of it